org.scala-ide.sdt.aspects/plugin.xml

```diff
@@ -8,4 +8,5 @@
    <extension-point id="cuprovider" name="Compilation Unit Provider" schema="cuprovider.exsd"/>
    <extension-point id="imagedescriptorselector" name="Image Descriptor Selector" schema="imagedescriptorselector.exsd"/>
    <extension-point id="openactionprovider" name="Open Action Provider" schema="openactionprovider.exsd"/>
+   <extension-point id="lncheck" name="Line Number Check" schema="lncheck.exsd"/>
```

Comment:
What was the point of this extension point? Was it an experiment? Do we no longer need it and can we squash the commits?